build/android/bytecode/java/org/brave/bytecode/BraveClassVisitor.java

```diff
@@ -147,6 +155,28 @@ protected void makePublicMethod(String className, String methodName) {
         methods.add(methodName);
     }
 
+    private String shouldChangeOwner(String owner, String methodName) {
```

Comment:
since the semantics of this are slightly different than the making methods public, maybe call it `maybeChangeOwner` and return the original owner instead of an empty string if it doesn't change? Then you don't need this https://github.com/brave/brave-core/pull/5199/files#diff-125eb2dc484493790df1e4b6cca9fd3bR77 and you can add the println inside `shouldChangeOwner`. Just a suggestion though